geozero/src/csv/csv_reader.rs

```diff
@@ -109,17 +111,14 @@ pub fn process_csv_geom(
     let geometry_idx = headers
         .iter()
         .position(|f| f == geometry_column)
-        .ok_or(GeozeroError::ColumnNotFound)?;
+        .ok_or(CsvError::ColumnNotFound)?;
 
     let mut collection_started = false;
 
     for (record_idx, record) in reader.into_records().enumerate() {
         let record = record?;
-        let geometry_field = record
-            .get(geometry_idx)
-            .ok_or(GeozeroError::ColumnNotFound)?;
-        let wkt = wkt::Wkt::from_str(geometry_field)
-            .map_err(|e| GeozeroError::Geometry(e.to_string()))?;
```

Comment:
Here you've gotten read of a to_string (heap allocation), so good job, so I think this one makes sense.